opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -853,6 +926,11 @@ def __init__(
         self.span_processor = span_processor
         self.id_generator = id_generator
         self.instrumentation_info = instrumentation_info
+        self._limits = None
+
+    def _with_limits(self, limits: _Limits) -> "Tracer":
```

Comment:
What is this for? We can just internally set the `_limits` attribute, right?

Reply:
Yes, provider can set `_limits` on tracer after creating one but this reads better to me as it can be chained and tracer's internal structure/logic does not leak into provider. Happy to change though.